src/calibre/devices/jetbook/driver.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
appears to be duplicated

Reply:
I have no idea, I do not think this was my sed...

Reply:
Actually I think that may have been a bad merge, there.